spec/models/authenticator/amazon_spec.rb

```diff
@@ -1,79 +1,48 @@
-require "aws-sdk-v1"
+require 'aws-sdk'
 
 describe Authenticator::Amazon do
+  AWS_ROOT_USER_KEY = 'aws_root_key'
```

Comment:
Why constants? They look weird to me; is there precedent suggesting this is good rspec style?